src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -67,6 +66,17 @@ interface SleevePerson extends Person {
   storedCycles: number;
 }
 
+/** Various info about resets
+ * @public */
+interface ResetInfo {
+  /** Numeric timestamp (from Date.now()) of last augmentation reset */
+  lastAugReset: number;
+  /** Numneric timestamp (from Date.now()) of last node reset */
+  lastNodeReset: number;
+  /** The current node */
```

Comment:
I'm fine with calling these `lastNodeReset`/`currentNode` (i.e. being a bit coy in the naming, also it's a bit shorter).
However, I think in the comment we should actually call out that it is the current bitnode. If you don't know what it is yet, "node" and "bitnode" are equivalently opaque, but if you *do* know then "node" is a bit overloaded (Hacknet Node? BN? Something else?) whereas bitnode is unambiguous.